### PR TITLE
Also use cached favicon on breach detail page

### PR DESCRIPTION
**Pull request by Vinnl**
# References: 
Jira: MNTOR-1494
Figma: N/A

# Description

It'll be lower quality than the organic, free-range, hand-crafted logos served from our CDN, but at least it'll free up Maxx to learn some Qt. Meanwhile design can come up with a new design that doesn't feature the logos at this size.

# Screenshot (if applicable)

An ugly one:

image

[(Current)](https://monitor.firefox.com/breach-details/SlideTeam)

An OK one:

image

([Current](https://monitor.firefox.com/breach-details/FaceUP) uses a logo rather than an icon.)

Without a found favicon:

image

# How to test

Visit http://localhost:6060/breaches, then open a couple of individual breaches.

# Checklist (Definition of Done)
- [x] Localization strings (if needed) have been added.
- [x] Commits in this PR are minimal and [have descriptive commit messages](https://chris.beams.io/posts/git-commit/).
- [x] I've added or updated the relevant sections in readme and/or code comments
- [ ] I've added a unit test to test for potential regressions of this bug. - N/A
- [ ] Product Owner accepted the User Story (demo of functionality completed) or waived the privilege.
- [x] All acceptance criteria are met.
- [x] Jira ticket has been updated (if needed) to match changes made during the development process.
- [x] Jira ticket has been updated (if needed) with suggestions for QA when this PR is deployed to stage.

### CODE REVIEW

File: src/client/css/partials/breachDetails.css [hunk 1]
```diff
@@ -6,6 +6,21 @@
...
+  display: inline-block;
+}
+
+.breach-detail-header span.breach-logo {
+  line-height: 5rem;
+  width: 5rem;
+  border-radius: 5rem;
+  font-size: 3rem;
+  font-weight: bold;
+  text-transform: uppercase;
```
Comment (toufali) [inline src/client/css/partials/breachDetails.css:20]: Does `text-transform: uppercase;` have localization implications?  I recall some research @flodolo might have been part of – maybe this just works 🤞, or maybe we need to use `toLocaleUpperCase()`?

File: src/views/partials/breach-detail.js [hunk 2]
```diff
@@ -129,7 +130,7 @@ function makeBreachDetails (breach) {
 
 export const breachDetails = data => `
   <header class="breach-detail-header">
-    <img class="breach-detail-logo breach-logo" alt="" src="https://monitor.cdn.mozilla.net/img/logos/${data.breach.LogoPath}" />
+    ${getBreachLogo(data.breach, data.breachLogos)}
```
Comment (toufali) [inline src/views/partials/breach-detail.js:133]: Doesn't it feel great to reuse your recently refactored module/util? 😊

**Review by toufali — APPROVED** [review]
LGTM -- approving to unblock, but we might give Zachery and Flod a chance to respond to the open questions here and in Slack.

File: src/views/partials/breach-detail.js (hunk 2 above)
Comment (mansaj) [reply src/views/partials/breach-detail.js:133]: nice! :D

**Review by mansaj — APPROVED** [review]

**Review by rhelmer — APPROVED** [review]
lgtm!

File: src/client/css/partials/breachDetails.css (hunk 1 above)
Comment (flodolo) [reply src/client/css/partials/breachDetails.css:20]: Don't…
https://mozilla-l10n.github.io/documentation/localization/dev_best_practices.html#css-issues

Comment (flodolo) [reply src/client/css/partials/breachDetails.css:20]: One thing that I don't understand from this patch is what this text is? Is it the name of the breached company, we shouldn't make decision on how their name is capitalized.

File: src/client/css/partials/breachDetails.css [hunk 3]
```diff
@@ -6,6 +6,21 @@
   padding: var(--padding-xl) var(--padding-sm);
 }
 
+.breach-detail-header .breach-logo {
+  height: 5rem;
```
[the remark this answers is not in the document]
Comment (Vinnl) [reply src/client/css/partials/breachDetails.css:10]: Thanks for the clarification. I think in this case, like regular images, it's nice if it scales along with the fonts, to keep the same proportions compared to the title.

**Review by flodolo — APPROVED** [review]

**Pull request merged**